pkg/gardenlet/controller/shoot/shoot_control.go

```diff
@@ -671,7 +672,38 @@ func (r *shootReconciler) updateShootStatusOperationStart(ctx context.Context, g
 		LastUpdateTime: now,
 	}
 
-	return gardenClient.Status().Update(ctx, shoot)
+	var mustRemoveOperationAnnotation bool
+
+	switch shoot.Annotations[v1beta1constants.GardenerOperation] {
+	case v1beta1constants.ShootOperationRotateCAStart:
+		if shoot.Status.Credentials == nil {
+			shoot.Status.Credentials = &gardencorev1beta1.ShootCredentials{}
+		}
+		if shoot.Status.Credentials.Rotation == nil {
+			shoot.Status.Credentials.Rotation = &gardencorev1beta1.ShootCredentialsRotation{}
+		}
+		if shoot.Status.Credentials.Rotation.CertificateAuthorities == nil {
+			shoot.Status.Credentials.Rotation.CertificateAuthorities = &gardencorev1beta1.ShootCARotation{}
+		}
+		shoot.Status.Credentials.Rotation.CertificateAuthorities.Phase = gardencorev1beta1.RotationPrepare
+		mustRemoveOperationAnnotation = true
+
+	case v1beta1constants.ShootOperationRotateCAComplete:
+		shoot.Status.Credentials.Rotation.CertificateAuthorities.Phase = gardencorev1beta1.RotationComplete
```

Comment:
Add nil checks? We don't now, how messed around with the status and don't want gardenlet to panic...